src/components/active-students/ActiveStudentsBlock.tsx

```diff
@@ -18,18 +17,27 @@ const ActiveStudentsBlock = () => {
   const { t } = useTranslation()
   const navigate = useNavigate()
 
-  const getMyCooperations = useCallback(
-    () => cooperationService.getCooperations({ limit: 3, status: 'active' }),
-    []
-  )
+  const fetchMyCooperations = async (): Promise<
+    ItemsWithCount<Cooperation>
+  > => {
+    const response = await cooperationService.getCooperations({
+      limit: 3,
+      status: 'active'
+    })
+    return response.data as ItemsWithCount<Cooperation>
+  }
 
-  const { loading, response, error } = useAxios<ItemsWithCount<Cooperation>>({
-    service: getMyCooperations,
-    defaultResponse
+  const {
+    data = defaultResponse,
+    isLoading,
+    isError
+  } = useQuery<ItemsWithCount<Cooperation>, Error>({
```

Comment:
```suggestion
  } = useQuery({
```
It will be typed automatically after using `baseService`